### PR TITLE
[WBCAMS-447] fix indexing of benefits

### PR DESCRIPTION
**Pull request by jonathan-longe**

### CODE REVIEW

File: src/WorkBC.ElasticSearch.Indexing/Services/XmlParsingServiceFederal.cs [hunk 1]
```diff
@@ -309,26 +312,67 @@ public ElasticSearchJob ConvertToElasticJob(string federalXml, bool isFrench = f
...
-                                benefit = benefit
-                                    .Replace("Rrsp", "RRSP")
-                                    .Replace("Resp", "RESP");
+                                benefit = "RRSP benefits";
+                            }
+                            
+                            if (benefit.Contains("resp"))
+                            {
+                                benefit = "RESP benefits";
+                            }
```
Comment (jonathan-longe) [inline src/WorkBC.ElasticSearch.Indexing/Services/XmlParsingServiceFederal.cs:330]: In the XML, the registered savings plan benefits are spelled out in full with the abbreviation at the end like this:  "Registered retirement savings plan (rrsp)"

**Comment by infojunkie** [comment]
Thanks @jonathan-longe I am asking @sunanditasahu to review the PR because she's more familiar with the code than I am.

File: src/WorkBC.ElasticSearch.Indexing/Services/XmlParsingServiceFederal.cs (hunk 1 above)
Comment (jonathan-longe) [reply src/WorkBC.ElasticSearch.Indexing/Services/XmlParsingServiceFederal.cs:330]: @sunanditasahu thanks for the review!  You raise a good point, I too wondered about the logic. However, the following line:  ` var benefit = option.InnerText.Capitalize();` changes the entire string to lowercase except the first letter and the logic thereafter can be certain of the string capitalization.

Comment (jonathan-longe) [reply src/WorkBC.ElasticSearch.Indexing/Services/XmlParsingServiceFederal.cs:330]: I realize that my comment above didn't show the correct capitalization from the XML.

> In the XML, the registered savings plan benefits are spelled out in full with the abbreviation at the end like this: "Registered retirement savings plan (rrsp)"

The following would have been more accurate:

> In the XML, the registered savings plan benefits are spelled out in full with the abbreviation at the end like this: "Registered Retirement Savings Plan (RRSP)"

Comment (jonathan-longe) [reply src/WorkBC.ElasticSearch.Indexing/Services/XmlParsingServiceFederal.cs:330]: @sunanditasahu would you feel more comfortable if I changed the if / then logic to include comparisons that are clearly case insensitive or should I leave it as is?

**Pull request merged**